src/QsCompiler/BondSchemas/Execution/Execution.cs

```diff
@@ -83,43 +83,38 @@ public enum ResultValue
 
     [global::Bond.Schema]
     [System.CodeDom.Compiler.GeneratedCode("gbc", "0.12.1.0")]
-    public partial class ArrayValue
+    public partial class Argument
```

Comment:
Without values, and with its use in the Entry Point Operations, could we rename this type to `Parameter` instead? The term `argument` is more of a value that is bound to a parameter at run time, where as the term `parameter` is more related to the definition of the callable.

Reply:
Sure, that makes sense to me. I'll keep the name `ArgumentValue` as is, though, to make it very clear that it has a value (even though it is technically implicit)

Reply:
Thanks! That will make it easier for me to reason about this.